src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+
+CrUX is a public dataset of real-user experiences in Chrome. It powers some of the critical tools
+in the Core Web Vitals workflow like [Search Console](https://support.google.com/webmasters/answer/9205520#about_data)
+and [PageSpeed Insights](https://developers.google.com/speed/docs/insights/v5/about#crux).
+The raw data for millions of websites is also publicly queryable in the [CrUX dataset on BigQuery](https://web.dev/chrome-ux-report-bigquery)
+and the [CrUX API](https://web.dev/chrome-ux-report-api). For this update to the Web Vitals
+extension, we've integrated it with the page and origin-level desktop data in the CrUX API. User
+experience data is broken down into three qualitative ratings: good, needs improvement, and poor.
+The thresholds used for each rating are documented in the guides for each of the Core Web Vitals
+metrics: [LCP](https://web.dev/lcp/#what-is-lcp), [FID](https://web.dev/fid/#what-is-fid), and
+[CLS](https://web.dev/cls/#what-is-a-good-cls-score). So for example, the CrUX API can tell us that
+16% of real-user experiences on a given page are evaluated as having good
+<span style="white-space: nowrap;">LCP ≤ 2.5</span> seconds.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/EaVNEuQ6gxVLtHYNkSZj.png", alt="Screenshot of the
+Web Vitals extension showing an explanation of how the local LCP experience relates to real-user
+desktop data from the field.", width="800", height="549" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing an explanation of how the local LCP experience
+    relates to real-user desktop data from the field.
+  </figcaption>
+</figure>
+
+The way this is integrated with the Web Vitals extension is by layering your local experience on
+top of the broader distribution of real-user experiences. It's like when you're in an expansive
+place like a shopping mall and there's a big map with all of the stores and a pin labeled
+"YOU ARE HERE" to help you orient yourself and find where you're going. The distributions of
+real-user data from CrUX are laid out for each metric and your local experiences are horizontally
+positioned in such a way that indicates where your experiences fall in relation to the
+distributions. Something that should be immediately apparent from this UI is when your local
+experience is very different from other users. For example, if you have a slow LCP experience and
+only 1% of users have similar experiences, something unusual must have happened.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/t5SmXUJqJadNsyOi26bb.png", alt="Screenshot of the
+Web Vitals extension showing origin-level desktop field data with \"Waiting for input…\" for the
+FID results.", width="800", height="552" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing origin-level desktop field data with
+    "Waiting for input…" for the FID results.
+  </figcaption>
+</figure>
+
```

Comment:
Good place for a subheading. Something like "Displaying page-level field data when origin-level data is not available"